w3-blob.md

```diff
@@ -380,15 +426,23 @@ type BlobPut = {
 
 ### Put Blob Subject
 
-The subject field SHOULD be [`did:key`] corresponding to the [Ed25519] private key that is first 32 bytes of the blob [multihash].
+The subject field SHOULD be [`did:key`] corresponding to the [Ed25519] private key that is last 32 bytes of the blob [multihash].
```

Comment:
@vasco-santos I have changed this to last 32 bytes because that is actual sha-256 of the content while first 32 will include multihash prefix thing which will be always the same.